packages/manager/src/components/TransferDisplay/TransferDisplayDialog.tsx

```diff
@@ -0,0 +1,137 @@
+import { useTheme } from '@mui/material/styles';
+import * as React from 'react';
+
+import { Box } from 'src/components/Box';
+import { Dialog } from 'src/components/Dialog/Dialog';
+import { Divider } from 'src/components/Divider';
+import { Link } from 'src/components/Link';
+import { Typography } from 'src/components/Typography';
+
+import { TransferDisplayUsage } from './TransferDisplayUsage';
+import { NETWORK_TRANSFER_QUOTA_DOCS_LINKS } from './constants';
+import { getDaysRemaining } from './utils';
+
+import type { RegionTransferPool } from './utils';
+
+export interface TransferDisplayDialogProps {
+  generalPoolUsage: PoolUsage;
+  generalPoolUsagePct: number;
+  isOpen: boolean;
+  onClose: () => void;
+  regionTransferPools: RegionTransferPool[];
+}
+
+type PoolUsage = {
+  quota: number;
+  used: number;
+};
+
+export const TransferDisplayDialog = React.memo(
+  (props: TransferDisplayDialogProps) => {
+    const {
+      generalPoolUsage: { quota, used },
+      generalPoolUsagePct,
+      isOpen,
+      onClose,
+      regionTransferPools,
+    } = props;
+    const theme = useTheme();
+    const daysRemainingInMonth = getDaysRemaining();
+
+    const transferQuotaDocsText =
+      used === 0
+        ? 'Compute instances, NodeBalancers, and Object Storage include network transfer.'
+        : 'View products and services that include network transfer, and learn how to optimize network usage to avoid billing surprises.';
+
+    return (
+      <Dialog
+        fullWidth
+        maxWidth="sm"
+        onClose={onClose}
+        open={isOpen}
+        title="Monthly Network Transfer Pool"
+      >
+        {/**
+         *  Global Transfer Pool Display
+         */}
+        <Typography
+          data-testid="general-transfer-pool-display"
+          fontFamily={theme.font.bold}
+          marginBottom={theme.spacing()}
+        >
+          Global Network Transfer Pool
+        </Typography>
+        <TransferDisplayUsage
+          pullUsagePct={generalPoolUsagePct}
+          quota={quota}
+          used={used}
+        />
+        <Divider
+          sx={{ marginBottom: theme.spacing(2), marginTop: theme.spacing(3) }}
+        />
+        {/**
+         *  Region-specific Transfer Pool Display
+         */}
+        {quota > 0 && regionTransferPools.length > 0 && (
```

Comment:
```suggestion
        {regionTransferPools.length > 0 && (
```
Was there a reason that we were checking `quota > 0` here? This results in the MNTP dialog not showing transfer data if a user has regional transfer in, for example, only Jakarta and no transfer in the global transfer pool, which is a plausible situation.

Reply:
you're right! Should have revisited after actually understanding the feature 🙄

Reply:
done